AppProject/Client/Pages/Homepage.razor

```diff
@@ -0,0 +1,9 @@
+﻿@page "/home"
+@using AppProject.Client.Pages.Components;
+@using AppProject.Client.Shared;
+
+<PageTitle>Games</PageTitle>
+
+@code {
```

Comment:
Das `@code { }` kann weg, da Homepage.razor.cs Datei vorhanden ist

Reply:
Homepage würde ich evtl drin lassen, oder soll ich die rausnehmen falls du in einem anderen PR schon eine andere datei hast?!
würde den PR dann evl erst dienstag abend abschließen.

Reply:
Die Datei kann gerne da bleiben :). Mir ging es hier nur um den `@code{ }` Teil in der 7ten Zeile, da dieser Code Abschnitt schon in eine eigene Datei ausgelagert ist.